01-node-tutorial/answers/customEmitter.js

```diff
@@ -0,0 +1,23 @@
+//Set up our own event with variable 
+// by requiring event module
+const EventEmitter = require('events');
+
+//Create instance of our class/ object
+const customEmitter = new EventEmitter();
+
+//Register a listener(callback function) for 'response'(event name)
+customEmitter.on('response',() =>{
```

Comment:
To answer your question: "Why for timer example we can emit first and use on method after?"

It has to do with the order of execution of your code. In Node, the sequence in which events are emitted and listeners are registered matter. You can emit an event first and register a listener afterward, but the listener won’t catch any events emitted before its registration. In your setInterval where events are emitted repeatedly, registering the listener afterward works because the listener will catch subsequent events.

Reply:
thank you!